### DISCUSSION TITLE
add new column to existing table, then cant able to access the table anymore.

### DISCUSSION THREAD

**Discussion by Div-47** [simolus3/drift#1337] [Q&A]
plz help me out. i heard about migrations but i dont want to migrate. only want to add a new column to existing table.

**Comment by davidmartos96** [comment]
If you are developing the app, you can safely delete the database so that it is recreated with the new columns. If otherwise, that version of the database is out in the wild with users using it, you need to handle that migration accordingly.

**Reply by Div-47** [reply]
can u plz guide how to delete db and recreate it again. 
am new to offline and i find moor best than other . plz

**Reply by davidmartos96** [reply]
It depends on how you are creating the database. If you are using `VmDatabase` you should see where it is located.

**Reply by simolus3** [reply]
You can also go to system settings and delete all data of your app in there.

**Reply by Div-47** [reply]
i am using ios simulator, and i have uninstalled the app, delete the moor_generated.g.dart file  and regenerate the moor_genereted.g.dart file again then run, then i am not able to see the data present previuosly  in my ui , rest of data are showing as it is. 
only the problem is where i altered the table.

**Reply by simolus3** [reply]
When you delete a database, of course all previous data will be lost. The new database will have the right columns though. If you don't want to delete the database, you need to write a migration.
